weed/filer/filer_deletion.go

```diff
@@ -141,14 +147,36 @@ func (f *Filer) deleteChunksIfNotNew(oldEntry, newEntry *Entry) {
 
 	var toDelete []*filer_pb.FileChunk
 	newChunkIds := make(map[string]bool)
-	for _, newChunk := range newEntry.Chunks {
+	newDChunks, newMChunks, err := ResolveChunkManifest(f.MasterClient.GetLookupFileIdFunction(),
```

Comment:
please spell out the variable names.  saving a few characters in `newDChunks newMChunks` is not worth it.

Reply:
I just followed the convention like:
https://github.com/chrislusf/seaweedfs/blob/da3d330616663e1c843514bd7a7a2c1833bf42d1/weed/filer/filechunk_manifest.go#L71
https://github.com/chrislusf/seaweedfs/blob/da3d330616663e1c843514bd7a7a2c1833bf42d1/weed/shell/command_volume_fsck.go#L156
https://github.com/chrislusf/seaweedfs/blob/da3d330616663e1c843514bd7a7a2c1833bf42d1/weed/shell/command_volume_fsck.go#L335

Should I change these `dChunks` to `dataChunks` together? @chrislusf

Reply:
yes, please.

Reply:
done @chrislusf